algorithms/sock_merchant/kevin/kevin.py

```diff
@@ -1,17 +1,28 @@
-#!/bin/python3
+#!usr/bin/env python3
 # Kevin Boyette
 import sys
```

Comment:
Probably don't need this anymore either

Reply:
He does if he wants to run it from command line without prefixing with python

Reply:
@AlexLanzano I meant the `import sys` line not the shebang

Reply:
@prithajnath @AlexLanzano I cleaned it up a bit more.  Thanks for catching the leftover import!

Reply:
Oh, Im stupid. Carry on